README.md

```diff
@@ -51,30 +51,35 @@ Run the following command to lint all python scripts:
 
 * `flake8`
 
-## Utility Functions
+## Model Experiments
 
-All utility functions are in `utils` directory.
+We performed experiments with a number of existing models listed below to understand how useful they are in helping
+with generating new Bliss symbols etc.
 
-### Scale down images (utils/scale_down_images.py)
+### StyleGAN3
 
-This script scales down JPG and PNG images in a directory to a specified size while maintaining their aspect ratios. 
-The output images are saved in a new directory. If the output directory doesn't exist, it will be created.
+Conclusion: not useful
 
-**Usage**: python scale_down_images.py [input_dir] [output_dir] [new_size]
+See the [TrainStyleGAN3MOdel.md](./docs/TrainStyleGAN3MOdel.md) in the [documentation](../docs) folder for details
```

Comment:
Broken link -- `./docs/TrainStyleGAN3MOdel.md` --  the "O" in "Model" should be lower case "o".